{"name": "Получение заметки по некорректному ID", "status": "passed", "steps": [{"name": "Попытка получения заметки с некорректным ID=!@#$", "status": "passed", "start": 1759765432928, "stop": 1759765432930}], "parameters": [{"name": "invalid_id", "value": "'!@#$'"}], "start": 1759765432927, "stop": 1759765432930, "uuid": "3aa83ef2-e0c3-40fe-b6c6-57565dd8da9f", "historyId": "43df5bdf2ec361a70b80c5cba592b932", "testCaseId": "efeb17e83461f5117735527552790e0e", "fullName": "tests.test_api.TestNoteRetrieval#test_get_note_by_invalid_id", "labels": [{"name": "story", "value": "Получение заметок"}, {"name": "feature", "value": "Управление заметками"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestNoteRetrieval"}, {"name": "host", "value": "E4435"}, {"name": "thread", "value": "8244-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}], "titlePath": ["tests", "test_api.py", "TestNoteRetrieval"]}